raytracing/matrix.py

```diff
@@ -1095,6 +1103,14 @@ def drawAt(self, z, axes, showLabels=False):  # pragma: no cover
                               clip_on=True)
         axes.add_patch(p)
 
+    def transferMatrix(self, upTo=float('+Inf')):
```

Comment:
You need to return a Matrix (or MatrixGroup) that has DielectricInterface (infinite radius) + Space(appropriate length).